content/applications/hr/employees/new_employee.rst

```diff
@@ -193,7 +193,9 @@ types needed.
           :align: center
           :alt: A skill form for a Math skill type, with all the information entered.
 
-Work Information tab
+.. _employees/work-info-tab:
+
+Work information tab
```

Comment:
It was correct to accept this incoming change to add the custom anchor, however we want to preserve the 16.0 version's heading, which has a capital "I" in _Information_ :)
```suggestion
Work Information tab
```

Reply:
Ahhh, ok- I saw the conflict listing the anchors being the issue, so I thought by accepting the merge conflict, it was taking everything from the forward ported doc just not the anchors for some reason.  So that's why I tried to resolve it the way I did.